pkg/manifest/anaconda_installer_iso_tree.go

```diff
@@ -594,12 +596,29 @@ func (p *AnacondaInstallerISOTree) bootcInstallerKickstartStages() []*osbuild.St
 	// Because osbuild core only supports a subset of options, we append to the
 	// base here with some more hardcoded defaults
 	// that should very likely become configurable.
-	hardcodedKickstartBits := `
-reqpart --add-boot
+	var rootFsType string
+	switch p.InstallRootfsType {
+	case "btrfs":
+		rootFsType = "btrfs"
+	case "xfs":
+		rootFsType = "xfs"
+	default:
+		rootFsType = "ext4"
+	}
```

Comment:
I'd prefer to not do a default inside `images` and to instead handle the default inside `bootc-image-builder` and `panic()`'ing here if an invalid `InstallRootfsType` is passed into it.